backend/controllers/productController.js

```
@@ -63,6 +68,85 @@
+        .status(400)
+        .json({ success: false, message: "Unsupported File" });
+    }
+
+    data = data.map((product) => {
+      if (product.bestSeller !== undefined) {
+        product.bestSeller =
+          product.bestSeller === "TRUE" || product.bestSeller === true;
+      }
+
+      if (product.date) {
+        product.date = new Date(parseInt(product.date));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)